generate-new-strings-pot.sh

```diff
@@ -52,23 +54,33 @@ if [[ "$CI_PULL_REQUEST" ]]; then
 	COMMITSURL=https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/${CI_PULL_REQUEST##*/}/commits
 	echo "LocalCI - fetching $FILESURL"
 	GH_FILESURL_CONTENT=$(auth_gh_curl $FILESURL)
+
+	# Disable exit on error, this section uses non-0 exit codes
+	set +o errexit
+
 	ANY_CHANGED_FILES=$(echo $GH_FILESURL_CONTENT | jq -r '.[] .filename' )
 	if [ $? -ne 0 ]; then
 	    echo "Error parsing $FILESURL:"
 	    echo $GH_FILESURL_CONTENT
 	fi
+
 	CHANGED_FILES=$(echo "$ANY_CHANGED_FILES" | grep -e '.jsx$' -e '\.js$' )
 	if [ $? -ne 0 ]; then
 	    echo "No JS files changed."
 	    exit 0
 	fi
+
 	echo "LocalCI - fetching $COMMITSURL"
 	GH_COMMITSURL_CONTENT=$(auth_gh_curl $COMMITSURL)
 	COMMITS_HASHES=$(echo $GH_COMMITSURL_CONTENT | jq -r '.[] .sha');
 	if [ $? -ne 0 ]; then
 	    echo "Error parsing $COMMITSURL:"
 	    echo $GH_COMMITSURL_CONTENT
```

Comment:
Same here, this also seems like a failure, but we print an error message and don't `exit 1` 😕

Reply:
IIRC this is because it still needs to call `move_pot_to_output` and `exit 3` later in the code.